Array2D map
geometry_msgs/Pose start
geometry_msgs/Pose goal
---
geometry_msgs/PoseStamped[] path